### PR TITLE
Fee Recipient Revision

### PR DESCRIPTION
**Pull request by symbolpunk**
This PR revises our Fee Recipient doc.

Metadoc: https://www.notion.so/prysmaticlabs/Metadoc-Fee-Recipient-afbe2d78e9944646a9aadc1c13645860

Preview: https://deploy-preview-420--prysmdocs.netlify.app/docs/next/execution-node/fee-recipient/

### CODE REVIEW

File: website/docs/execution-node/fee-recipient.md [hunk 1]
```diff
@@ -1,83 +1,172 @@
 ---
 id: fee-recipient
-title: Setting an Address for Receiving Transaction Fees
+title: How to configure Fee Recipient on your client instance and/or beacon node
```
Comment (symbolpunk) [inline website/docs/execution-node/fee-recipient.md:3]: How do we feel about normalizing the use of "How to...." in titles of docs that are how tos? Pros that I see:

- May improve discoverability via search when people type "how to _______" 
- May be a step in the direction of organizing our docs into "How Tos" vs "Concepts" vs "Walkthroughs" - how tos being a specific type of walkthrough. Current thinking on this topic -> https://www.notion.so/prysmaticlabs/Principles-Content-0b2944089c99400f8d1ee3138b24fb65#ff546df2fb964819abef67b45ca3e176

File: website/docs/execution-node/fee-recipient.md [hunk 2]
```diff
@@ -1,83 +1,172 @@
...
 sidebar_label: Setting an Address for Receiving Transaction Fees
+doc_type: how-to
+sme: james-prysm
 ---
 
-# Fee Recipient
 
-Validators don't currently receive any portion of the transaction fees paid by end-users when proposing new blocks.
+# How to configure Fee Recipient on your client instance and/or beacon node
```
Comment (symbolpunk) [inline website/docs/execution-node/fee-recipient.md:10]: Updating # H1 to make it clear to the reader that they're reading a how-to with a specific goal in mind, not a concept.

Comment (symbolpunk) [reply website/docs/execution-node/fee-recipient.md]: Reverting based on how clumsy it feels when rendered in large font, and based on the current pattern being followed in this node's sidebar.

File: website/docs/execution-node/fee-recipient.md (hunk 1 above)
Comment (symbolpunk) [reply website/docs/execution-node/fee-recipient.md]: Reverting based on how clumsy it feels when rendered in large font, and based on the current pattern being followed in this node's sidebar.

File: website/docs/execution-node/fee-recipient.md [hunk 3]
```diff
@@ -1,83 +1,192 @@
...
 
-**Fee Recipient** is a feature that addresses this. With Fee Recipient configured, validators will receive a portion of transaction fees when proposing blocks.
+# Configuring a Fee Recipient Address 
 
-This document shows you how to configure a custom address to receive transaction fees using your Prysm validator client software. 
+<!-- alt: ## How to configure Fee Recipient on your client instance and/or beacon node -->
 
-:::tip Post-Merge
-This feature will activate after [The Merge](https://ethereum.org/en/upgrades/merge/).
+**Fee Recipient** is a feature that lets you specify a priority fee recipient address on your validator client instance and beacon node. After [The Merge](https://ethereum.org/en/upgrades/merge/), execution clients will begin depositing priority fees into this address whenever your validator client proposes a new block.
```
Comment (james-prysm) [inline website/docs/execution-node/fee-recipient.md:15]: I wonder if there is a feature to link down to another section so you can explain users what a priortiy fee is. is this something we are using or is this standardized somewhere in existing documentation? 🤔

File: website/docs/execution-node/fee-recipient.md [hunk 4]
```diff
@@ -1,83 +1,192 @@
...
-`fee-recipient-config-url` flag requires this json format provided through url.
+#### Fee Recipient JSON Config File
 
-This will allow you to map your validators to corresponding eth addresses or generally cover the remaining keys with a default.
+If you use either `fee-recipient-config-file` or `fee-recipent-config-url` to specify your fee recipient address, your JSON configuration should follow this schema:
 
 ```
 {
   "proposer_config": {
     "0xa057816155ad77931185101128655c0191bd0214c201ca48ed887f6c4c6adf334070efcd75140eada5ac83a92506dd7a": {
```
Comment (james-prysm) [inline website/docs/execution-node/fee-recipient.md:96]: I have no idea if these fee recipients and addresses are real peop's accounts or not, wondering if we can placeholder them in a good way.

File: website/docs/execution-node/fee-recipient.md [hunk 5]
```diff
@@ -1,83 +1,192 @@
...
+</table>
+
+<br />
+
+
+### Configuring Fee Recipient on your beacon node
+
+A fee recipient address can be configured on your beacon node instance by using the `suggested-fee-recipient` flag.
+
+:::warning This is an additive flag
```
Comment (james-prysm) [inline website/docs/execution-node/fee-recipient.md:158]: all flags are additive we probably don't need this.

File: website/docs/execution-node/fee-recipient.md (hunk 3 above)
Comment (symbolpunk) [reply website/docs/execution-node/fee-recipient.md:15]: Clarifying -> are you recommending 

1) linking to a glossary definition that defines priority fee? (this is on my radar! but not within the scope of this PR.)
2) inline/in-doc definitions? (this strikes me as an optimization - step 1 define in glossary, step 2 embed from glossary)
3) linking to the explanation of priority fee a couple lines below this copy?

File: website/docs/execution-node/fee-recipient.md (hunk 4 above)
Comment (symbolpunk) [reply website/docs/execution-node/fee-recipient.md]: Great thought. I just pushed an update so all addresses start with `0x01234567` to make it clear that they're placeholders.

This pattern has been documented here: https://www.notion.so/prysmaticlabs/Patterns-13002df64d454c16b076151694491fc8

File: website/docs/execution-node/fee-recipient.md (hunk 5 above)
Comment (symbolpunk) [reply website/docs/execution-node/fee-recipient.md]: Removed.

File: website/docs/execution-node/fee-recipient.md (hunk 3 above)
Comment (james-prysm) [reply website/docs/execution-node/fee-recipient.md:15]: well at first I was thinking 1, but realized you described it in the background so i guess 2.

**Review by james-prysm — APPROVED** [review]
LGTM for now 👍

**Pull request merged**